src/main/java/com/epam/test_generator/dao/impl/CaseVersionDAOImpl.java

```diff
@@ -23,16 +44,21 @@
 
     @Override
     public List<CaseVersion> find(Long caseId) {
```

Comment:
please, rename this method to findForId()

Reply:
done